release.json

```diff
@@ -13,6 +13,22 @@
 		"JMXFETCH_VERSION": "0.33.0",
 		"JMXFETCH_HASH": "078011fbcca5e9beee2f041fefacbad824cff47e074b74a4674eb3efed8984b3"
 	},
+	"7.15.0-rc.3": {
+		"INTEGRATIONS_CORE_VERSION": "6.15.0-rc.3",
+		"OMNIBUS_SOFTWARE_VERSION": "6.15.0-rc.1",
```

Comment:
`omnibus-software` has been updated since this tag. So I think we should create a new tag on `omnibus-software` and update it here.

Reply:
Fixed